frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -370,12 +373,17 @@ class RegistrationForm extends Component {
               </div>
               <button
                 disabled={!submitButtonEnabled}
-                style={styles.loginBtn}
+                style={styles.createAccountBtn}
                 className="btn btn-primary"
                 type="submit"
               >
                 {LOCALIZE.authentication.createAccount.button}
               </button>
+              {!submitButtonEnabled && (
+                <div tabIndex="0" style={styles.enableButtonDesc}>
```

Comment:
Just curious: why does this need a tabIndex?

Reply:
Because, if a screen reader user thinks that all fields are valid (if it's not the case for example), the next tab entry after password confirmation will bring him to this message, so he'll know that he missed something, meaning that at least of the fields is invalid.

Reply:
Something like that:

![image](https://user-images.githubusercontent.com/23021242/58898970-d153f680-86c9-11e9-91af-882688e7dc6d.png)

Reply:
ahh, ok. Interesting.

Reply:
Wouldn't a screen reader read this without a tab index thought?